opensrp-chw/src/ba/assets/json.form/pnc_health_facility_visit_sw.json

```diff
@@ -167,6 +167,13 @@
         "read_only": "true",
         "hidden": "true",
         "value": "0"
+      },
+      {
+        "key": "pnc_hf_next_visit_date",
```

Comment:
Should you be using **anc_hf_next_visit_date** as the entity ID? This should be the **pnc_hf_next_visit_date**

Reply:
Fixed as mentioned above